### PR TITLE
New Add-on: Scan Rules Pack

### PR DESCRIPTION
**Pull request by psiinon**
Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Review by thc202 — APPROVED** [review]

File: addOns/packscanrules/packscanrules.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import org.zaproxy.gradle.addon.AddOnStatus
+
+description = "All of the add-ons just containing release, beta and alpha status scan rules"
+
+zapAddOn {
+    addOnName.set("Collection: Scripting Pack")
```
Comment (kingthorin) [inline addOns/packscanrules/packscanrules.gradle.kts:6]: Scan Rules

File: addOns/packscanrules/packscanrules.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+        dependencies {
+            addOns {
+                register("ascanrules")
+                register("ascanrulesAlpha")
+                register("ascanrulesBeta")
+                register("domxss")
+                register("pscanrules")
+                register("pscanrulesAlpha")
+                register("pscanrulesBeta")
```
Comment (kingthorin) [inline addOns/packscanrules/packscanrules.gradle.kts:22]: Retire?
JWT?
FileUpload?

File: addOns/packscanrules/packscanrules.gradle.kts (hunk 1 above)
Comment (psiinon) [reply addOns/packscanrules/packscanrules.gradle.kts]: Doh! Fixed

File: addOns/packscanrules/packscanrules.gradle.kts (hunk 2 above)
Comment (psiinon) [reply addOns/packscanrules/packscanrules.gradle.kts:22]: Added Retire.js
Not so sure about the other 2 - they are more manual and already in the Pantester Pack..

**Review by kingthorin — APPROVED** [review]

**Pull request merged**